apiserver/plane/api/serializers/issue.py

```
@@ -32,6 +33,26 @@
 from django.core.validators import URLValidator
 
 
+def is_uuid(value):
+    try:
+        uuid_obj = uuid.UUID(str(value))  # Convert to string in case it's not already
```

Ruff (0.8.2)

Local variable `uuid_obj` is assigned to but never used

Remove assignment to unused variable `uuid_obj`

(F841)